### PR TITLE
Remove magic numbers

### PR DESCRIPTION
**Pull request by andrewreal**
Replaced magic numbers with variables.

Issue: 40

### CODE REVIEW

**Comment by ritik210200** [comment]
Blank lines

**Review by MileyHollenberg — APPROVED** [review]
LGTM, no need to test locally since the values remain the same but are now just stored in a property

File: UOP1_Project/Assets/Scripts/Characters/Character.cs [hunk 1]
```diff
@@ -23,6 +24,7 @@ public class Character : MonoBehaviour
     private float verticalMovement = 0f; //Represents how much a player will move vertically in a frame. Affected by gravity * gravityContributionMultiplier
     private Vector3 inputVector; //Initial input horizontal movement (y == 0f)
     private Vector3 movementVector; //Final movement vector
+    private float rotationThreshold = .02f; // Used to prevent NaN result causing rotation in a non direction
```
Comment (m4rtint) [inline UOP1_Project/Assets/Scripts/Characters/Character.cs:27]: if it's not being changed on inspector, you can probably set it as a 
`private const float RotationThreshold`

Comment (ciro-unity) [reply UOP1_Project/Assets/Scripts/Characters/Character.cs:27]: Good point!

**Comment by ciro-unity** [comment]
Implemented @m4rtint's suggestion

**Comment by ciro-unity** [comment]
Close #40

**Pull request merged**